buildkite/resource_pipeline_test.go

```diff
@@ -581,9 +581,11 @@ func TestAccPipeline_remove_team(t *testing.T) {
 					// Quick check to confirm the local state is correct before we update it
 					resource.TestCheckResourceAttr("buildkite_pipeline.foobar", "name", "Test Pipeline foo"),
 				),
+				ImportStateVerifyIgnore: []string{"slug"},
```

Comment:
Is this what you were talking about at the standup?

What was the error showing up? I'm wondering if there is something else going wrong here and adding this is masking it

Reply:
Yes, this is what we discussed during standup yesterday. Interestingly, the test no longer fails locally if I remove `ImportStateVerifyIgnore`